.github/workflows/autolabel.yml

```diff
@@ -24,5 +24,17 @@ jobs:
                     "label": "marked for removal/rejection",
                     "status": "removed",
                     "path": "Formula/.+"
+                }, {
+                    "label": "bottle unneeded",
+                    "path": "Formula/.+",
+                    "content": "bottle :unneeded"
```

Comment:
If the workflow doesn't already do this: might want to make sure it's checking for whitespace, beginning/end of line characters around these to avoid matching in e.g. comments.

Reply:
`path` and `content` are actually regexes, so we can tweak it more as we like.